README.rst

```diff
@@ -25,6 +25,15 @@ Overview
 This package contains a utility to create a test Firefox Account on either the dev or prod instance of Firefox Accounts,
 as well as a set of page objects that can be used to interact with Firefox Accounts' sign in screens.
 
+Installation
+------------
+
+``python setup.py develop``
+
+**If running on a Mac, you may need the following before running the above command:**
+
+``pip install cryptography==1.2.1``
```

Comment:
PyFxA doesn't pin cryptography, so we probably shouldn't either: https://github.com/mozilla/PyFxA/blob/master/setup.py#L39. Let's change this to just `pip install cryptography`